backend/test_observer/main.py

```diff
@@ -18,14 +18,22 @@
 #        Omar Abou Selo <omar.selo@canonical.com>
 #        Nadzeya Hutsko <nadzeya.hutsko@canonical.com>
 
+import os
 
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
+import sentry_sdk
+
 
 from test_observer.controllers.router import router
 
-app = FastAPI()
 
+SENTRY_DSN = os.getenv("SENTRY_DSN")
+if SENTRY_DSN:
+    sentry_sdk.init(SENTRY_DSN)  # type: ignore
```

Comment:
Just in case, I had to tell mypy to ignore `Cannot instantiate abstract class "init" with abstract attribute "__exit__"  [abstract]` error while initialising sentry client. It's a [known bug](https://github.com/getsentry/sentry-python/issues/1415) that was fixed in later version. But since our self-hosted Sentry instance doesn't support latest clients that include this bug fix, we had to go with this approach.